app/Console/Commands/SyncBunny.php

```
@@ -171,6 +171,187 @@ private function syncReleasesToGitHubRepo(): bool
         }
     }
 
+    /**
+     * Sync both releases.json and versions.json to GitHub repository in one PR
+     */
+    private function syncReleasesAndVersionsToGitHubRepo(string $versionsLocation, bool $nightly = false): bool
```

PHPMD (2.15.0)

The method syncReleasesAndVersionsToGitHubRepo has a boolean flag argument $nightly, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)